manifests/rdoinfo.pp

```diff
@@ -17,6 +17,11 @@
     mode   => '0755',
     owner  => 'rdoinfo',
   } ->
+  exec { 'ensure home contents belong to rdoinfo':
+    command => 'chown -R rdoinfo:rdoinfo /home/rdoinfo',
```

Comment:
In this case we are setting rdoinfo's home mode to 0755, so if we set recurse => true it would force that permission into all files, which is not necessarily what we want.